security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolation.java

```diff
@@ -199,14 +259,67 @@ public double getValue() {
      * other than branches.
      *
      * @return the side of the equipment where the violation occurred. Will be {@code null} for equipments
-     * other than branches.
+     * other than branches and three windings transformers.
      */
     @Nullable
+    @Deprecated
```

Comment:
You should remove the use of this deprecated method (and use getBranchSide at lease where it is relevant in the core)